packages/usa-radio/src/usa-radio.stories.js

```diff
@@ -9,6 +9,11 @@ export default {
       control: { type: "radio" },
       options: ["none", "disabled", "aria-disabled"],
     },
+    indeterminate: {
+      name: "Indeterminate option (toggle to reset)",
```

Comment:
The reason I added it is because you can still select radio buttons while the `indeterminate` is turned on. Toggling the switch will then properly reset the `indeterminate` state.

Here's a quick demo:

[Kapture 2024-03-04 at 12.15.04.webm](https://github.com/uswds/uswds/assets/25211650/bc22bef5-cbb6-417d-9c41-913b79d19c98)